packages/vscode-extension/package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
 	"name": "suitecloud-vscode-extension",
-	"version": "1.3.0",
+	"version": "1.3.1",
```

Comment:
If we are going to update the vscode-extension later, I would not commit this file yet.

Reply:
Committed the wrong package-lock.json. Updated.